### DISCUSSION TITLE
整合包加入通用拼音搜索模组后崩溃

### DISCUSSION THREAD

**Discussion by Killua109** [ET-Team/EnigTech2#660] [Q&A] [answered]
下了个整合包加了一下最新版本的通用拼音搜索，随后报错，更换更老版本的通用拼音搜索模组后仍然报错，附两份崩溃报告
[crash-2022-10-07_21.49.08-client.txt](https://github.com/ET-Team/EnigTech2/files/9734534/crash-2022-10-07_21.49.08-client.txt)
[crash-2022-10-07_21.55.20-client.txt](https://github.com/ET-Team/EnigTech2/files/9734535/crash-2022-10-07_21.55.20-client.txt)

**Comment by dong031001** [comment, marked as answer]
有没有一种可能 包里本来就有…
